examples/cactus-example-carbon-accounting-frontend/src/zone-flags.ts

```diff
@@ -2,4 +2,5 @@
  * Prevents Angular change detection from
  * running with certain Web Component callbacks
  */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
@akaashsaxena2897 If you disable the linter please also attach an explanation detailing why it was not feasible/impossible to fix the underlying error instead of turning off the linter for the line. (if no such explanation exists then the recommended course of action is to fix the linter warning instead of turning off the linter)

Reply:
@akaashsaxena2897 Why not this?

```typescript
type WindowWithZone = { __Zone_disable_customElements: boolean } & Window &
  typeof globalThis;

/**
 * Prevents Angular change detection from
 * running with certain Web Component callbacks
 */
(window as WindowWithZone).__Zone_disable_customElements = true;
```

Reply:
Thanks, I think that's a great approach as well, let me implement it

Reply:
Hey, I have made the changes as requested, if there's no other issue then I'd like to request you to review and merge the code